tests/ship_streamer_test.py

```diff
@@ -97,20 +97,26 @@
     cluster.biosNode.kill(signal.SIGTERM)
 
     Print("Create a jumbo row")
+    jumboAcc = Account("itsjumbotime")
+    jumboAcc.ownerPublicKey = "EOS6MRyAjQq8ud7hVNYcfnVPJqcVpscN5So8BhtHuGYqET5GDW5CV"
+    jumboAcc.activePublicKey = "EOS6MRyAjQq8ud7hVNYcfnVPJqcVpscN5So8BhtHuGYqET5GDW5CV"
+    nonProdNode.createAccount(jumboAcc, cluster.eosioAccount)
```

Comment:
Was there some reason a new account was needed?

Reply:
For some reason I could no longer push an action with `defproducerb`. It wasn't clear why; my assumption was something has changed wrt keys for producers in the test harness due to IF that prevents it from singing transactions easily. Possibly a mixup wrt producer key vs account key.

A run with the failure is at
https://github.com/AntelopeIO/spring/actions/runs/9360819344/job/25766994628
```
2024-06-04T03:51:56.6085969Z 2024-06-04T03:48:14.434310 Create a jumbo row
2024-06-04T03:51:56.6087951Z 2024-06-04T03:48:14.434482  cmd: /__w/spring/spring/build/bin/cleos --url http://localhost:8892  --wallet-url http://localhost:9899 --no-auto-keosd -v set contract -j -f  defproducerb unittests/contracts/jumborow jumborow.wasm jumborow.abi
2024-06-04T03:51:56.6090555Z 2024-06-04T03:48:14.460290  ERROR: Exception during set contract. stderr: Reading WASM from /__w/spring/spring/build/unittests/contracts/jumborow/jumborow.wasm...
2024-06-04T03:51:56.6091818Z Publishing contract...
2024-06-04T03:51:56.6092665Z [31mError 3090003: Provided keys, permissions, and delays do not satisfy declared authorizations[0m
2024-06-04T03:51:56.6093968Z [32mEnsure that you have the related private keys inside your wallet and your wallet is unlocked.[0m
2024-06-04T03:51:56.6094899Z [33mError Details:
2024-06-04T03:51:56.6096505Z transaction declares authority '{"actor":"defproducerb","permission":"active"}', but does not have signatures for it.[0m
```

Reply:
The wallet is not populated in this test, so the only valid key is the default bios node key which is what you used here for jumboAcc. Which also is the default eosio key. Ideally Cluster.py should have a getter for this, but it seems we just hard code it in all the existing tests.

Reply:
I guess `defproducerb`'s account keys have been changed then

> Ideally Cluster.py should have a getter for this, but it seems we just hard code it in all the existing tests.

yep I felt back copy pasting it in but I had noticed that was the MO elsewhere too